Sources/ViewInspector/SwiftUI/ViewThatFits.swift

```diff
@@ -0,0 +1,24 @@
+import SwiftUI
+
+@available(iOS 16.0, macOS 13.0, tvOS 16.0, *)
+internal extension ViewType {
+    struct ViewThatFits {}
+}
+
+@available(iOS 16.0, macOS 13.0, tvOS 16.0, *)
+extension ViewType.ViewThatFits: SingleViewContent {
+
+    static func child(_ content: Content) throws -> Content {
+        let view: Any = try {
+            guard let rootContent = try? Inspector.attribute(path: "_tree|content", value: content.view) else {
+                // A ViewThatFits View renders only one of its child Views based on the available horizontal space.
```

Comment:
```suggestion
                // A ViewThatFits View renders only one of its child Views based on the available space.
```

Reply:
The view accepts an axis on initialization.
<img width="914" alt="Screenshot 2023-08-23 at 10 39 31 AM" src="https://github.com/nalexn/ViewInspector/assets/1791049/094e453e-d60d-4d04-bf02-be55dcb50740">